src/git_changelog/build.py

```diff
@@ -36,19 +89,15 @@ def bump(version: str, part: str = "patch") -> str:  # noqa: WPS231
     Returns:
         The bumped version.
     """
-    prefix = ""
-    if version[0] == "v":
-        prefix = "v"
-        version = version[1:]
-
-    semver_version = VersionInfo.parse(version)
+    semver_version = PrefixedVersionInfo.parse(version)
     if part == "major" and semver_version.major != 0:
         semver_version = semver_version.bump_major()
     elif part == "minor" or (part == "major" and semver_version.major == 0):
         semver_version = semver_version.bump_minor()
     elif part == "patch" and not semver_version.prerelease:
         semver_version = semver_version.bump_patch()
-    return prefix + str(semver_version)
+
+    return str(semver_version)
```

Comment:
The lines you removed here are so much easier to read and understand than the class you suggest :confused: